strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/artifact/coordinates/MockedMavenArtifactCoordinates.java

```diff
@@ -49,45 +49,6 @@ public MockedMavenArtifactCoordinates()
         defineCoordinates(GROUPID, ARTIFACTID, VERSION, CLASSIFIER, EXTENSION);
     }
 
-    public MockedMavenArtifactCoordinates(String path)
-    {
-        this(ArtifactUtils.convertPathToArtifact(path));
-    }
-
-    public MockedMavenArtifactCoordinates(String... coordinateValues)
```

Comment:
I know this was ugly, but what was the to remove this constructor?

Reply:
Not used probably, will revert it back